endpoints/cookie_sync_test.go

```diff
@@ -329,13 +385,24 @@ func TestCookieSyncHandle(t *testing.T) {
 			metrics:         &mockMetrics,
 			pbsAnalytics:    &mockAnalytics,
 			accountsFetcher: &fakeAccountFetcher,
+			time:            &fakeTime{time: time.Now()},
```

Comment:
Please use a specific time instead of `time.Now()` for tests. Any random date will do. Same comment for the other instances.

Reply:
used fixed value `time: &fakeTime{time: time.Date(2024, 2, 22, 9, 42, 4, 13, time.UTC)},`